frontend/packages/knative-plugin/src/components/add/EventSourceForm.tsx

```diff
@@ -32,11 +34,13 @@ const EventSourceForm: React.FC<FormikProps<FormikValues> & OwnProps> = ({
     <EventSourcesSelector eventSourceList={useEventSourceList()} />
     {values.type === EventSources.CronJobSource && <CronJobSection />}
     {values.type === EventSources.SinkBinding && <SinkBindingSection />}
+    {values.type === EventSources.KafkaSource && <KafkaSourceSection />}
     <SinkSection namespace={namespace} />
     <AppSection
       project={values.project}
       noProjectsAvailable={projects?.loaded && _.isEmpty(projects.data)}
     />
+    <AdvancedSection />
```

Comment:
am not sure if we need `<AdvancedSection />` always  as none of the other sources has so maybe can have only for `kafkaSource` or shall we show limits (resources/requests) in main form itself 

cc @christianvogt @serenamarie125

Reply:
If none of the other sources are going to have any advanced options then we can show it only for kafka